### DISCUSSION TITLE
Supabase jwt secret CLI

### DISCUSSION THREAD

**Discussion by Fnux8890** [supabase/supabase#26767] [Questions]
I am using the supabase cli with init, link and start. when I start it it says that my jwt secret is the default `super-secret-jwt-token-with-at-least-32-characters-long` 
- How do I change it?
- Why isn't it the same as the supabase project when I link?

I can find nowhere about how to fix this or that it even is possible. if it is not possible then why display it to me in the cli tool?

**Comment by tommedema** [comment]
Same question here, have been looking for an answer for several days but doesn't seem to be anywhere in the docs
